### ISSUE TITLE
Update fluent-bit version in sumologic-kubernetes-collection helm chart

### ISSUE THREAD

**Issue by amitkc1** [SumoLogic/sumologic-kubernetes-collection#2107]
**Describe the bug**
 The default fluent-bit version used with sumologic-kubernetes-collection helm chart (v2.1.5) is at v0.12.1. This may contain security vulnerabilities. This image relies on end of Life Debian OS as a base. Hence, this should be upgraded to a newer version.

**Logs**
Logs which can be helpful in investigating the issue.

**Command used to install/upgrade Collection**
SUMOLOGIC_CHART_VERSION=2.1.5

helm repo add sumologic https://sumologic.github.io/sumologic-kubernetes-collection
echo "Fetching sumologic-kubernetes-collection chart"
helm fetch \
  --untar \
  --untardir ./charts \
  --version ${SUMOLOGIC_CHART_VERSION} \
    sumologic/sumologic

helm template \
  collection --namespace sumologic \
  -f values.yaml \
  ./charts/sumologic > backup/04-sumologic-collection.yaml

**Configuration**
Values.yaml does not modify fluent-bit version.

**To Reproduce**
Install sumologic using 2.1.5 helm chart version. You can use default values.

**Expected behavior**
Fluent-bit version should be up to date/ relatively newer than v0.12.1

**Environment (please complete the following information):**
 - NA
 - NA 
 - NA
 - NA
**Anything else do we need to know**
As mentioned this is not exactly a bug. But, I've marked bug because this is an issue for due to possible security vulnerabilities. Also, based on what I've come to know this fluent-bit image relies on end of Life Debian OS as a base.

**Comment by kkujawa-sumo** [comment]
We tried to update Fluent Bit version but after observing instability (gaps in logs, logs were not delivered) this change was reverted (#1824, #1863).
Recently we rebuilt Fluent Bit 1.6.10 to update dependencies, this version is available in container registry: `public.ecr.aws/sumologic/fluent-bit:1.6.10-sumo-1` and it comes from https://github.com/SumoLogic/fluent-bit-docker-image

To use `public.ecr.aws/sumologic/fluent-bit:1.6.10-sumo-1` you need modify configuration in `values.yaml`:

```yaml
fluent-bit:
  image:
    repository: public.ecr.aws/sumologic/fluent-bit
    tag: 1.6.10-sumo-1
```

**Comment by amitkc1** [comment]
Thank you. This image definitely has less vulnerabilities. It will be good to have this image version reflected in sumologic-kubernetes-collection parent helm chart. Because, even if we change the image this way, multiple fluent-bit resources will have incorrect labels on them. This includes, serviceAccount, clusterRoleBinding etc. Some of them can be overridden via the values.yaml. For example, service or pod labels can be overridden. But, there is a good chance that this approach will create discrepancy between the actual image used and the version indicated via the kubernetes resource label.

**Comment by amitkc1** [comment]
For the purpose of my original question/issue, I think this can be considered resolved.

**Comment by perk-sumo** [comment]
> It will be good to have this image version reflected in sumologic-kubernetes-collection parent helm chart

It will be included `v2.5.2` release, should be out soon.
